tests/formatter_tests.ts

```diff
@@ -119,7 +121,7 @@ function buildTickContexts(): void {
 }
 
 function rightJustify(options: TestOptions): void {
-  const f = VexFlowTests.makeFactory(options, 1200, 300);
+  const f = VexFlowTests.makeFactory(options, 1200, 150);
```

Comment:
why did you change this value?

Reply:
Trying to conserve vertical space.  Will it mess with the visual regression?

Reply:
That is ok I thought that it might have been overseen.